### PR TITLE
extend LDAP config options

### PR DESCRIPTION
**Pull request by SISheogorath**
add Default for Trusted Certificates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ docker run -p 443:443 -e LDAP_URI="ldap://ldap.example.com/dc=example,dc=com?uid
 |`DHPARAM_PEM`|yes|| Apache's SSL/TLS PEM DHParameter."
 |`CUSTOM_APACHE_CONFIG`|no||Specifies custom parameters to be appended to the apache virtualhost configuration.|
 |`LDAP_URI`|yes||Specifies the URI of the LDAP server, as documented [here](<https://httpd.apache.org/docs/2.4/mod/mod_authnz_ldap.html#authldapurl> "Apache docs").|
+|`LDAP_BIND_DN`|no||Specifies the DN used for search, as documented [here](<https://httpd.apache.org/docs/2.4/mod/mod_authnz_ldap.html#authldapbinddn> "Apache docs").|
```
Comment (dariko) [inline README.md:26]: There should be a note about the exclusivity between the `LDAP_BIND_DN` and `LDAP_BIND_USER_PATTERN` parameters

**Review by dariko — CHANGES_REQUESTED** [review]
The SSL default cert configuration is ok.
Setting both apache's `AutlLDAPBindDN` and `AuthLDAPInitialBindPattern` parameters would be inconsistent.
This container should refuse to run if both `LDAP_BIND_USER_PATTERN` and `LDAP_BIND_DN` are set: there should be a  check in /run.sh and a note in README.md

**Comment by dariko** [comment]
Thank you, I've added the notes/checks about the exclusivity contraint between `LDAP_BIND_DN` - `LDAP_BIND_USER_PATTERN` and merged.

**Comment by SISheogorath** [comment]
Thanks!

**Pull request merged**